packages/elements/src/components/theme-switch/theme-switch.scss

```diff
@@ -13,7 +15,7 @@ $width: 2.9em;
     }
   } @else if $exp < 0 {
     @for $i from 1 through -$exp {
-      $value: $value / $number;
+      $value: math.div($value, $number);
```

Comment:
Why is this included here?

Reply:
I noticed some warnings from the sass compiler because its new version won't support the `/` syntax anymore. So I ran the migration script for it. I can revert the changes if you want